docs_src/microservices/core/command/Ch-Command.md

```diff
@@ -432,7 +432,7 @@ The payload is the base64-encoding json struct:
 2. Receive command response message from external MQTT broker on topic `edgex/command/response/#`
 ```json
 {
-  "ReceivedTopic":"edgex/device/command/response/device-virtual/Random-Boolean-Device/Bool/set",
+  "ReceivedTopic":"edgex/response/device-virtual/e6e8a2f4-eb14-4649-9e2b-175247911369",
```

Comment:
The ReceivedTopic, aka Response topic, on external MQTT will be `edgex/command/response/Random-Boolean-Device/WriteBoolValue/set`

See https://github.com/edgexfoundry/edgex-go/blob/main/internal/core/command/controller/messaging/external.go#L129